maas-api/internal/tier/handler_test.go

```diff
@@ -231,7 +232,8 @@ func TestHandler_PostTierLookup_DisplayNameFallback(t *testing.T) {
 	}
 
 	clientset := fake.NewClientset([]runtime.Object{configMap}...)
-	mapper := tier.NewMapper(t.Context(), clientset, fixtures.TestTenant, fixtures.TestNamespace)
+	testLogger := logger.New(false) // Use production logger for tests
```

Comment:
`logger.New(false)` not sure how to read it. If we stick with our wrapper maybe we can have dedicated funcs like `logger.Production()` and `logger.Debug()`?

Reply:
Done! Added `logger.Production()` and `logger.Development()` factory functions for better readability. `New(bool)` is still available but marked as "prefer using Production()/Development()".

Reply:
We don't need to deprecate for backward compatibility - it's a new thing.